pgjdbc/src/test/java/org/postgresql/test/jdbc2/ResultSetTest.java

```diff
@@ -1152,4 +1161,50 @@ private Map<String, Integer> getResultSetColumnNameIndexMap(ResultSet stmt) {
     return null;
   }
 
+  private static class CallableWithConnection implements Callable<Boolean> {
+
+    private final Connection connection;
+    private final int expectedYear;
+
+    protected CallableWithConnection(Connection connection, int expectedYear) {
+      this.connection = connection;
+      this.expectedYear = expectedYear;
+    }
+
+    @Override
+    public Boolean call() {
+      try (Statement statement = connection.createStatement()) {
+        for (int i = 0; i < 10; i++) {
+          try (ResultSet resultSet = statement.executeQuery(
+              String.format("SELECT unnest(array_fill('8/10/%d'::timestamp, ARRAY[%d]))",
+                  expectedYear, 500))) {
+            while (resultSet.next()) {
+              Timestamp d = resultSet.getTimestamp(1);
+              int year = 1900 + d.getYear();
+              if (year != expectedYear) {
+                return false;
+              }
+            }
+          }
+        }
+      } catch (SQLException sqlException) {
+        fail(sqlException.getMessage());
+        return false;
+      }
```

Comment:
Can you just propagate exception?
`fail(...)` would hide the stacktrace

Reply:
done